### ISSUE TITLE
Solr extension's cellar

### ISSUE THREAD

**Issue by RSully** [Homebrew/homebrew-php#2467]
My Cellar is not in `/usr/local` and when I installed `php56-solr` it gave me the following problem:

```
PHP Warning:  PHP Startup: Unable to load dynamic library '/.../homebrew/opt/php56-solr/solr.so' - dlopen(/.../homebrew/opt/php56-solr/solr.so, 9): Library not loaded: /.../homebrew/opt/libxml2/lib/libxml2.2.dylib
  Referenced from: /.../homebrew/opt/php56-solr/solr.so
  Reason: Incompatible library version: solr.so requires version 12.0.0 or later, but libxml2.2.dylib provides version 10.0.0 in Unknown on line 0
```

Reinstalling with `--build-from-source` fixed it. 

I noticed that 56-solr has `cellar :any` and the others (53, 54, 55) all are `:any_skip_relocation`. I'm not sure what these 2 mean specifically, but it sounds like it may be related.

**Comment by ablyler** [comment]
@RSully interesting, want to submit a pull request that changes that line?

**Comment by ablyler** [comment]
@RSully you can also edit that line locally in your formula to see if it fixes it first

**Comment by RSully** [comment]
It looks like that line was authored by @BrewTestBot in both formulae. Any idea if it is possible to get it to re-generate to see if it still does that? I'm not sure how it chooses which to use.

**Comment by ablyler** [comment]
@mikemcquaid any input on this?

**Comment by mikemcquaid** [comment]
@ablyler Unfortunately not, it's autodetected but the detection isn't 100% reliable (such as this case).

**Comment by ablyler** [comment]
@mikemcquaid should we change the line manually in that one formula? will that stick?

**Comment by mikemcquaid** [comment]
@ablyler I'm not sure. You might be best adding a comment and saying it needs to be done manually.

**Comment by ablyler** [comment]
Should be fixed by PR #2494
